frontend/javascripts/oxalis/model/accessors/dataset_accessor.js

```diff
@@ -103,6 +103,40 @@ export function getElementClass(dataset: APIDataset, layerName: string): Element
   return getLayerByName(dataset, layerName).elementClass;
 }
 
+export function getDefaultIntensityRangeOfLayer(
+  dataset: APIDataset,
+  layerName: string,
+): [number, number] {
+  const maxFloatValue = 3.40282347e38;
+  const maxDoubleValue = 1.79769313486232e308;
+  const elementClass = getElementClass(dataset, layerName);
+  switch (elementClass) {
+    case "uint8":
+      return [0, Math.pow(2, 8) - 1];
+    case "uint16":
+      return [0, Math.pow(2, 16) - 1];
+    case "uint24":
+      return [0, Math.pow(2, 24) - 1];
+    case "uint32":
+      return [0, Math.pow(2, 32) - 1];
+    case "uint64":
+      return [0, Math.pow(2, 64) - 1];
+    // We do not fully support signed int data;
+    case "int16":
+      return [0, Math.pow(2, 15) - 1];
+    case "int32":
+      return [0, Math.pow(2, 31) - 1];
+    case "int64":
+      return [0, Math.pow(2, 63) - 1];
+    case "float":
+      return [0, maxFloatValue];
```

Comment:
@philippotto I reopened a PR for the better default intensityRange values.

Strangely, the screenshot tests do not fail for me (locally with `yarn test-screenshot`).
Nevertheless, I am pretty sure why the screenshot tests failed: The previous default intensityRange for float was `[0;255]`. With this PR the max value will become much larger, causing the data to be black.

The core issue here, in my opinion, is that we do did not fully agree on a "good" default value for floating-point data types.  
The high max value of `3.40282347e38` only makes sense when this is the actual max value for most of the float data layers. 
I think almost all float data layers have the range of [0, 255], thus we should treat float types the way we did before.
I hope you get my point, but let's talk about this in the office.